components/form/index.tsx

```diff
@@ -1,64 +1,57 @@
 import {
-  // Box,
+  Stack,
   FormControl,
   FormLabel,
   FormErrorMessage,
   FormHelperText,
   Input,
+  InputGroup,
+  InputLeftAddon,
   Button,
   Textarea,
   Select,
-  // Checkbox,
   Radio,
   RadioGroup,
+  Checkbox,
+  Text,
 } from '@chakra-ui/react'
 import { Formik, Field, Form } from 'formik'
-import * as yup from 'yup'
 import Router from 'next/router'
 
-const schema = yup.object().shape({
-  title: yup.string().required(), // TODO: length validation
-  content: yup.string().required(),
-  postal: yup.string().required(), // TODO: need to do postal FSA validation
-  category: yup.string(), // TODO: Figure out enum validation for list options
-  // anonymous: yup.bool().required(),
-  // contact: yup.bool().required(),
-  // name: yup.string(),
-  // email: yup.string().email(),
-  // phone: yup.string(), // TODO: Phone validation
-  // twitter: yup.string(),
-})
+import storySchema, { TITLE_CHAR_LIMIT, STORY_WORD_LIMIT } from '../../lib/storySchema'
 
 type LoginFormInputs = {
   title: string
   content: string
   postal: string
-  category?: string
-  // anonymous: boolean
-  // contact: boolean
-  // name?: string
-  // email?: string
-  // phone?: string
-  // twitter?: string
+  category: string
+  anonymous: string
+  contact: boolean
+  name?: string
+  email?: string
+  phone?: string
+  twitter?: string
+  consent: boolean
 }
 
 const initialValues = {
   title: '',
   content: '',
   postal: '',
   category: '',
-  anonymous: '',
-  contact: '',
+  anonymous: 'true',
```

Comment:
This initial value is a string because the radio button expects that. I transform it to `boolean` on the way out.